helpers/utils.js

```diff
@@ -1,7 +1,10 @@
 const RED = 0xE74C3C;
+const CACHE_DIR = "./.cache";
```

Comment:
Should specify the cache directory in `config.json.` Also name the variable something more descriptive, something like song_cache_dir